app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import mongoose from '../services/mongoose.js';
+
+const name = 'inspiration'
+
+// using the one csv in the repo for now.
+// later, let this be variable.
+const file_path = `./assets/decks/${name}/${name}.csv`
+
+export const rows = await csv().fromFile(file_path);
+
+const schema_data = {}
+
+for (var header in rows[0]) {
```

Comment:
It's rather unusual to define a schema like this dynamically. The point of using Mongoose to define a schema at all is to get some degree of consistency out of Mongo documents. You'll want to make each field explicit here, as a safety check against what's contained in the CSVs. All fields can be optional, but they should at least be enumerated statically.